src/app/core/services/auth.service.ts

```diff
@@ -28,10 +29,15 @@ export class AuthService {
 
     getUserInfo(): User {
         const userInfo = localStorage.getItem("userInfo");
```

Comment:
usually, special service to work with local storage is created. It will let all the developers and application as a library users to use local storage in the same way. For example enum for keys can be created or data may be typed or some other logic may be incapsulated in the service